### PR TITLE
Profiles: Support emoji avatars for ens owners w/o ens avatar

### PR DESCRIPTION
**Pull request by benisgold**
Fixes TEAM2-61, TEAM2-25

## What changed (plus any additional context for devs)
- change "use" to "edit" in intro screen, if profile has already been modified
- allow users with ens profile but no ens avatar to pick an avatar emoji
- remove support for non-ens avatar images

## PoW (screenshots / screen recordings)
https://user-images.githubusercontent.com/15272675/174150506-0eedabf3-afcf-4ce5-8a87-8af50eb52509.mp4

## Final checklist

- [x] Assigned individual reviewers?
- [x] Added labels?
- [ ] Added e2e tests? if not please specify why
- [ ] If you added new files, did you update the CODEOWNERS file?

### CODE REVIEW

File: src/components/transaction-list/TransactionList.js [hunk 1]
```diff
@@ -271,9 +270,6 @@ export default function TransactionList({
     e => {
       const { selection } = e.nativeEvent;
       switch (selection) {
-        case 'newimage':
-          onAvatarChooseImage();
```
Comment (jxom) [inline src/components/transaction-list/TransactionList.js:275]: we are removing this completely now? has this been agreed on?

Comment (estebanmino) [reply src/components/transaction-list/TransactionList.js:275]: i agree with jake, don't think we should be removing this one

**Comment by benisgold** [comment]
ty for feedback, i redid this pr and uploaded a new demo

File: src/components/discover-sheet/RegisterENSSection.tsx [hunk 2]
```diff
@@ -27,6 +31,13 @@ export default function RegisterENSSection() {
   const { navigate } = useNavigation();
   const { colors } = useTheme();
   const { isReadOnlyWallet } = useWallets();
+  const { uniqueDomain } = useAccountENSDomainGroups();
+
+  useEffect(() => {
+    if (uniqueDomain?.name) {
+      prefetchENSProfileRecords({ name: uniqueDomain.name });
```
Comment (jxom) [inline src/components/discover-sheet/RegisterENSSection.tsx:38]: wat dis?

Comment (benisgold) [reply src/components/discover-sheet/RegisterENSSection.tsx:38]: in the ens intro screen, the text "Edit notben.eth" vs "Use notben.eth" depends on if notben.eth has any records set yet. I'm prefetching the records on the discover sheet so there's no lag when determining the text usage when first rendering the ens intro screen

File: src/hooks/useAccountENSDomainGroups.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import { useMemo } from 'react';
+import useAccountENSDomains from './useAccountENSDomains';
+import useAccountProfile from './useAccountProfile';
+
+export default function useAccountENSDomainGroups() {
```
Comment (jxom) [inline src/hooks/useAccountENSDomainGroups.ts:5]: maybe we should just put this stuff in `useAccountENSDomains` to prevent hook fatigue?

**Review by jxom — COMMENTED** [review]
when i go to my gf's ENS (read only), I can only see cancel. Should we remove the action sheet entirely? 

Screen Shot 2022-06-21 at 8 36 08 am

When I create a new wallet (from my own wallet), I see the incorrect actions:

Screen Shot 2022-06-21 at 8 38 06 am

when I clear the app cache, the correct options are shown again, but when I navigate back to my ENS wallet it shows the actions on the non-ENS wallet ("Choose from Library"/"Pick an Emoji".

**Comment by benisgold** [comment]
@jxom 

1. good catch on aleisha.eth
2. why do you think those are the incorrect options?

**Review by estebanmino — APPROVED** [review]
looking good! 🔥 

making a comment that this PR won't `remove support for non-ens avatar images
` from the description from what i see, right?

**Pull request merged**